### ISSUE TITLE
Compatibility with This Rocks!?

### ISSUE THREAD

**Issue by Dronicode** [Globox1997/EarlyStage#26]
### Make sure you are not opening a duplicate.

- [X] I have searched the issue tracker and did not find an issue describing my bug/feature.

### Description of your suggestion.

I also use another mod- This Rocks! https://modrinth.com/mod/this-rocks, because it also adds a lot of nice ground decorations. EarlyStage does not seem to add its own rocks and flints to some modded biomes, but the other one does. It would be awesome to have some sort of compatibility where pairs of stone splitters from This Rocks! could be combined to craft rocks for EarlyStage. 

I made a small datapack to handle this for myself with tags but its very rudimentary and already improved the experience for me so I thought to share the idea because it might be useful for other players too.

**Comment by ender1324** [comment]
yeah rocks dont seem to appear in modded biomes which makes the mod unusable with biome mods

Edit: it seems to work with biomes o plenty but not oh the biomes you'll go and Terralith

**Comment by Globox1997** [comment]
Added compat with [https://github.com/Globox1997/EarlyStage/commit/4f024fbadc5bae7e2f1066b3abb37ee2b6425d71](https://github.com/Globox1997/EarlyStage/commit/4f024fbadc5bae7e2f1066b3abb37ee2b6425d71)
